cryptofeed/exchange/okcoin.py

```diff
@@ -181,6 +189,30 @@ async def _book(self, msg: dict, timestamp: float):
                 if self.checksum_validation and self.__calc_checksum(pair) != (update['checksum'] & 0xFFFFFFFF):
                     raise BadChecksum
                 await self.book_callback(self.l2_book[pair], L2_BOOK, pair, False, delta, timestamp_normalize(self.id, update['timestamp']), timestamp)
+                
+    
+    async def _order(self, msg: dict, timestamp: float):
+        if msg['data'][0]['status'] == "open":
+            status = "active"
+        else:
+            status = msg['data'][0]['status']
+
+        keys = ('filled_size', 'size', 'price', 'filled_notional')
+        data = {k: Decimal(msg['data'][0][k]) for k in keys if k in msg['data'][0]}
+
+        await self.callback(ORDER_INFO, feed=self.id,
+                            symbol=symbol_exchange_to_std(msg['data'][0]['instrument_id'].upper()),  # This uses the REST endpoint format (lower case)
+                            status=status,
+                            order_id=msg['data'][0]['order_id'],
+                            side=BUY if msg['data'][0]['side'].lower() == 'buy' else SELL,
+                            order_type=msg['data'][0]['order_type'],
+                            timestamp=msg['data'][0]['timestamp'].timestamp(),
+                            receipt_timestamp=timestamp,
+                            **data
+                            )
+        
+    async def _login(self, msg: dict, timestamp: float):
+        print(f"Login result: {msg}")
```

Comment:
No print statements. If you want, you can log this as info or debug (there are logging examples throughout)

Reply:
Removed in second commit